mbuild/compound.py

```diff
@@ -80,6 +83,25 @@ def load(filename, relative_to_module=None, compound=None, coords_only=False,
              "distances and standard residue templates!")
         structure = pmd.load_file(filename, structure=True, **kwargs)
         compound.from_parmed(structure, coords_only=coords_only)
+
+    elif smarts:
+        pybel = import_('pybel')
```

Comment:
Could you create a custom message that will be printed if this fails? This would go into `mbuild.utils.io`. See [this example](https://github.com/mosdef-hub/mbuild/blob/61f621d9f691bfe98d56af34733db45a8df5ff8f/mbuild/utils/io.py#L54-L64).